### PR TITLE
Xdmod7.0 ak fixes

### PR DESCRIPTION
**Pull request by nsimakov**
Several bug fixes.
1) cron job was pointing to wrong location for report generator and no cronjob for ingestion
2) AK view didn't unmask "Loading..." mask in case of no AK data.
3) Added missing html/controllers/data_explorer/* files, probably was left somewhere during migration

### CODE REVIEW

**Comment by nsimakov** [comment]
in AK button 'load defaults' was renamed to 'reset selection' and moved to reflect what it is actually doing

File: build.json [hunk 1]
```diff
@@ -1,6 +1,6 @@
 {
     "name": "xdmod-appkernels",
-    "version": "7.0.0",
+    "version": "7.0.1",
```
Comment (plessbd) [inline build.json]: is this going to be released as 7.0.1 or 7.1.0?

File: html/controllers/data_explorer/common.php [hunk 2]
```diff
@@ -0,0 +1,41 @@
...
+		return isset($_REQUEST['expandedAppKernels'])  && $_REQUEST['expandedAppKernels'] != '' ? explode(',',$_REQUEST['expandedAppKernels']):array();
+	}
+	
+	function getShowChangeIndicator()
+	{
+		return  ( isset($_REQUEST['show_change_indicator']) 
+        	      ? $_REQUEST['show_change_indicator'] === 'y' : false );
+	}
+
+?>
```
Comment (plessbd) [inline html/controllers/data_explorer/common.php]: add new line to end of file

File: html/controllers/data_explorer/get_tree.php [hunk 3]
```diff
@@ -0,0 +1,181 @@
...
+			}
+			$returnData[] = array('text' => $app_kernel->name,/*.' '.date('Y-m-d',$app_kernel->start_ts).' '.date('Y-m-d', $app_kernel->end_ts)*/
+				'id' => 'app_kernel_'.$app_kernel->id, 
+				'qtip' => $app_kernel->description,
+				'start_ts' => $app_kernel->start_ts,
+				'end_ts' => $app_kernel->end_ts,
+				'type' => 'app_kernel',
+				'iconCls' => 'appkernel', 
+				'leaf' => false,
+				//'singleClickExpand' => true,
```
Comment (plessbd) [inline html/controllers/data_explorer/get_tree.php]: remove commented out code

File: html/controllers/data_explorer/get_tree.php [hunk 4]
```diff
@@ -0,0 +1,181 @@
...
+	throw $see;
+}
+catch(Exception $ex)
+{
+	print_r($ex);
+	$returnData = array();
+}
+
+xd_controller\returnJSON($returnData);
+?>
```
Comment (plessbd) [inline html/controllers/data_explorer/get_tree.php]: add new line

**Pull request merged**